packages/react-native/Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -175,17 +175,21 @@ class TouchableOpacity extends React.Component<Props, State> {
       onLongPress: this.props.onLongPress,
       onPress: this.props.onPress,
       onPressIn: event => {
-        this._opacityActive(
-          event.dispatchConfig.registrationName === 'onResponderGrant'
-            ? 0
-            : 150,
-        );
+        if (!Platform.isTV) {
+          this._opacityActive(
+            event.dispatchConfig?.registrationName === 'onResponderGrant'
+              ? 0
+              : 150,
+          );
+        }
         if (this.props.onPressIn != null) {
           this.props.onPressIn(event);
         }
       },
       onPressOut: event => {
-        this._opacityInactive(250);
+        if (!Platform.isTV) {
```

Comment:
TODO: make sure this distinguishes correctly between touch events and TV remote control events.

Reply:
Done